includes/class-wc-mondido-admin-actions.php

```diff
@@ -28,13 +30,18 @@ public static function add_meta_boxes() {
 		global $post_id;
 		$order = wc_get_order( $post_id );
 		if ( $order && strpos( $order->get_payment_method(), 'mondido' ) !== false ) {
-			$transaction = get_post_meta( $order->get_id(), '_mondido_transaction_data', TRUE );
+			$orderStorage = OrderStorageTechnology::current();
+			$transaction = $orderStorage->get_meta( $order, '_mondido_transaction_data', TRUE );
 			if ( ! empty( $transaction ) ) {
+				$screen = class_exists( '\Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController' ) && wc_get_container()->get( CustomOrdersTableController::class )->custom_orders_table_usage_is_enabled()
```

Comment:
Is the call to `CustomOrdersTableController:: custom_orders_table_usage_is_enabled ` the same as `OrderUtil:: custom_orders_table_usage_is_enabled`? If so, I think we should use `OrderUtil` here as well.

Reply:
I've changed it in accordance to their documentation: https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book